packages/connect-query/src/utils.ts

```diff
@@ -56,25 +56,26 @@ export const isAbortController = (input: unknown): input is AbortController => {
  */
 export type ConnectUpdater<O extends DescMessage> =
   | MessageInitShape<O>
-  | ((prev?: MessageShape<O>) => MessageInitShape<O> | undefined);
+  | undefined
+  | ((prev?: MessageShape<O>) => MessageShape<O> | undefined);
```

Comment:
Adding `undefined` on line 59 supports a use-case with `QueryClient.setQueryData`: 

> If the value is undefined, the query data is not updated. 

This means the following example compiles and works as expected (query data is not updated):

```ts
const queryKey = createConnectQueryKey({
  schema: ElizaService.method.say,
  input: { sentence: "hi" },
  transport: myTransport,
});

const newData: SayResponse | undefined = undefined;

const updater = createProtobufSafeUpdater(
  ElizaService.method.say,
  newData,
);

queryClient.setQueryData(queryKey, updater);
```